sympy/printing/tests/test_lambdarepr.py

```diff
@@ -206,8 +206,17 @@ def test_numexpr():
     from sympy.logic.boolalg import ITE
     expr = ITE(x > 0, True, False, evaluate=False)
     assert NumExprPrinter().doprint(expr) == \
-           "evaluate('where((x > 0), True, False)', truediv=True)"
-
+           "numexpr.evaluate('where((x > 0), True, False)', truediv=True)"
+
+    from sympy.codegen.ast import Return, FunctionDefinition, Variable, Assignment
+    func_def = FunctionDefinition(None, 'foo', [Variable(x)], [Assignment(y,x), Return(y**2)])
+    print("")
+    print(NumExprPrinter().doprint(func_def))
+    expected = "def foo(x):\n"\
+               "    y = numexpr.evaluate('x', truediv=True)\n"\
+               "    return numexpr.evaluate('y**2', truediv=True)"
+    print(expected)
```

Comment:
Unfortunately I can't fix this right now due to being on a holiday. If someone else can pick this up it would be convenient. The line can just be removed.